<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>First</class>
 <widget class="QDialog" name="First">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="LabelFirst">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your first name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonFirstSubmit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonFirstReset">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditFirst">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferUppercase</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
